Content.Server/Storage/Components/EntityStorageComponent.cs

```diff
@@ -365,6 +364,10 @@ protected virtual IEnumerable<EntityUid> DetermineCollidingEntities()
         }
     }
 
+    public sealed class InsertIntoStorageAttemptEvent : CancellableEntityEventArgs
+    {
+
+    }
```

Comment:
actually rename this to `InsertIntoEntityStorageAttemptEvent` cause i got confused by that and assumed this was for regular storage earlier

Reply:
You got it boss!